modules/c++/math/include/math/Round.h

```diff
@@ -0,0 +1,73 @@
+/* =========================================================================
+ * This file is part of math-c++
+ * =========================================================================
+ *
+ * (C) Copyright 2004 - 2015, MDA Information Systems LLC
+ *
+ * math-c++ is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this program; If not,
+ * see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#ifndef __MATH_ROUND_H__
+#define __MATH_ROUND_H__
+
+#include <cmath>
+
+namespace math
+{
+/*!
+ *  Returns the nearest integral value towards zero.
+ *
+ *  \param value A number to evaluate
+ *  \return The 'fixed' number
+ */
+template<typename T> inline T fix(T value)
```

Comment:
Looks like it's called `fix` because that's what Matlab calls it: http://www.mathworks.com/help/matlab/ref/fix.html.  I would suggest leaving it as-is.